packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -165,6 +165,136 @@ const deskLocaleStrings = {
+  /** The tool tipe for the split pane button on the document panel header */
+  'document-panel-header.split-pane-button.tooltip': 'Split pane right',
+
+  /** The text content for the deleted document banner */
+  'banners.deleted-document-banner.text': 'This document has been deleted.',
+
+  /** The text for the restore button on the deleted document banner */
+  'banners.deleted-document-banner.restore-button.text': 'Restore most recent version',
+
+  /** The text for the reference change banner if the reason is that the reference has been changed */
+  'banners.reference-changed-banner.reason-changed.text':
+    'This reference has changed since you opened it.',
+
+  /** The text for the reload button */
+  'banners.reference-changed-banner.reason-changed.reload-button.text': 'Reload reference',
+
+  /** The text for the reference change banner if the reason is that the reference has been deleted */
+  'banners.reference-changed-banner.reason-removed.text':
+    'This reference has been removed since you opened it.',
+
+  /** The text for the close button */
+  'banners.reference-changed-banner.reason-removed.close-button.text': 'Close reference',
+
+  /** The text for the permission check banner if there is only one role */
+  'banners.permission-check-banner.singular-role.text':
+    'Your role {{roles}} does not have permissions to {{requiredPermission}} this document.',
+
+  /** The text for the permission check banner if there is are multiple roles */
+  'banners.permission-check-banner.plural-roles.text':
+    'Your roles {{roles}} do not have permissions to {{requiredPermission}} this document.',
+
+  /** The text for when a form is hidden */
+  'document-view.form-view.form-hidden': 'This form is hidden',
+
+  /** The text for when the form view is loading a document */
+  'document-view.form-view.loading': 'Loading document',
+
+  /** The title of the sync lock toast on the form view */
+  'document-view.form-view.sync-lock-toast.title': 'Syncing document…',
+
+  /** The description of the sync lock toast on the form view */
+  'document-view.form-view.sync-lock-toast.description':
+    'Please hold tight while the document is synced. This usually happens right after the document has been published, and it should not take more than a few seconds',
+
+  /** The title of the reconnecting toast */
+  'panes.document-pane-provider.reconnecting-toast.title': 'Connection lost. Reconnecting…',
+
+  /** The loading message for the document not found pane */
+  'panes.document-pane.document-not-found.loading': 'Loading document…',
+
+  /** The title of the document not found pane if the schema is known */
+  'panes.document-pane.document-not-found.title': 'The document was not found',
+
+  /** The text of the document not found pane if the schema is known */
+  'panes.document-pane.document-not-found.text':
+    'The document type is not defined, and a document with the <Code>{{id}}</Code> identifier could not be found.',
```

Comment:
did we merge the change that allowed for basic tags? if so can these now be changed to `<code>{{id}}</code>`?

Reply:
we have not merged that yet. When that goes in, I'll loop back and update these